nodewatcher/modules/devices/tplink/cpe.py

```diff
@@ -90,22 +90,22 @@ class TPLinkCPE210(cgm_devices.DeviceBase):
     }
 
 
-class TPLinkCPE220(TPLinkCPE210):
+class TPLinkCPE220v1(TPLinkCPE210v1):
     """
-    TP-Link CPE210 device descriptor.
+    TP-Link CPE210 v1 device descriptor.
```

Comment:
Fix this description (it was incorrect before as it is for CPE220 not CPE210).

Reply:
Fixed